crates/winch/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "winch"
```

Comment:
I think there are two different ways we could go here: a `winch` crate in its own subdirectory, as `cranelift` is, or a `wasmtime-winch` crate in `crates/winch` with a version number synchronized with the rest of the Wasmtime crates. Given the dependency on `wasmtime-environ` below I think it's probably more properly considered part of the Wasmtime family of crates, but if eventually factor that out, then it at least in theory could be its own thing usable separately (in the way that Cranelift is).

but tl;dr, a crate in `crates/$NAME` is part of Wasmtime and is named as `wasmtime-$NAME`...

Reply:
Yeah this is a good idea, I've completed this and I'm happy how things are shaping up.